control-plane/tests/test_phase17_boot_path_validation.py

```
@@ -0,0 +1,167 @@
+                            "AEGISOPS_CONTROL_PLANE_POSTGRES_DSN="
+                            "postgresql://aegisops_control_plane:secret@postgres:5432/"
+                            "aegisops_control_plane"
+                        ),
+                        "AEGISOPS_CONTROL_PLANE_POSTGRES_PASSWORD=secret",
+                        "",
+                    )
+                ),
+                encoding="utf-8",
+            )
+
+            services_result = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)